### PR TITLE
Add zkSync to `nft.transfers`

### PR DESCRIPTION
**Pull request by hildobby**

### CODE REVIEW

File: models/nft/zksync/nft_zksync_transfers_legacy.sql [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{{ config(
```
Comment (aalan3) [inline models/nft/zksync/nft_zksync_transfers_legacy.sql:1]: This file can just be a dummy.

File: models/nft/chains/nft_zksync_transfers.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        tags = ['dunesql'],
+        schema = 'nft_zksync',
+        alias =alias('transfers'),
+        partition_by=['block_month'],
+        materialized='incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_hash', 'evt_index', 'token_id', 'amount'],
+        pre_hook='{{ enforce_join_distribution("PARTITIONED") }}'
```
[the remark this answers is not in the document]
Comment (hildobby) [reply models/nft/chains/nft_zksync_transfers.sql:11]: sure sg, just copied the spell from another chain here but I'll remove it

**Review by aalan3 — APPROVED** [review]

File: models/_sector/nft/transfers/legacy/nft_zksync_transfers_legacy.sql
Comment (jeff-dude) [inline models/_sector/nft/transfers/legacy/nft_zksync_transfers_legacy.sql:1]: you can remove the legacy file now 🙏

File: models/_sector/nft/transfers/legacy/nft_transfers_legacy.sql
Comment (jeff-dude) [inline models/_sector/nft/transfers/legacy/nft_transfers_legacy.sql:1]: you can remove this modification, we will remove all existing legacy files soon

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**